src/components/editor/Bindings/Backfill/useUpdateBackfillCounter.ts

```diff
@@ -112,6 +125,14 @@ function useUpdateBackfillCounter() {
                                     backfilled && increment === 'false';
 
                                 if (shouldIncrement || shouldDecrement) {
+                                    increment === 'true'
```

Comment:
Since we need to handle this logic in two places... I think we could just store off which array to populate by storing the key up above before we start looping.